src/cluster.c

```diff
@@ -5839,16 +6025,19 @@ NULL
         /* SETSLOT 10 NODE <node ID> */
         int slot;
         clusterNode *n;
+        int reply = 1;
```

Comment:
I think this an artifact from the earlier revision?

Reply:
Good catch. Removed